doc/content/documentation/array-table-and-struct.md

```diff
@@ -84,19 +101,24 @@ To remove multiple values from arrays, the `remove` and `slice` functions can be
 (slice @[1 2 3 4] 2 1) # Evaluates to @[3]
 ```
 
+It is not possible to remove values from a set.
+
 ## Count elements
 
-To count the number of element of an array or table, the `count` function can be used.
+To count the number of element of an array, a table or a set, the `count` function can be used.
 
 ```phel
 (count @[1 2 3]) # Evaluates to 3
 (count @[]) # Evaluates to 0
 
 (count @{:a 1 :b 2}) # Evaluates to 2
 (count @{}) # Evaluates to 0
+
+(count (set 1 2 3)) # Evaluates to 3
+(count (set)) # Evaluates to 0
 ```
 
-## Array as a Stack
+## Array/Set as a Stack
```

Comment:
It may work that sets can be used as stack, but sets don't have a strict order. Therefore, from a functional view point they can not be used as stacks, because stacks required ordered data structures.